save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -126,16 +280,159 @@ class ProjectView : RComponent<ProjectProps, ProjectViewState>() {
         div("row") {
             child(cardComponent {
                 div("text-center") {
-                    div("pb-4") {
-                        h6(classes = "d-inline") {
-                            +"Path to property file: "
+                    div("card shadow mb-4") {
+                        a(classes = "d-block card-header py-3") {
+                            attrs {
+                                onClickFunction = { setState { isProjectAsGitUrl = !isProjectAsGitUrl } }
+                            }
+                            h6("m-0 font-weight-bold text-primary") {
+                                +"Upload project as git url"
+                            }
+                        }
+                        // Collapse card to load git url
+                        div {
+                            attrs.classes = if (state.isProjectAsGitUrl) setOf("collapse", "show") else setOf("collapse")
```

Comment:
How do you handle collapsing, after all? Maybe, it still would make sense to use `data-parent` instead of keeping track manually?